doc/man3/SSL_CTX_set_options.pod

```diff
@@ -217,6 +217,25 @@ not propose, and servers will not accept the extension.
 Disable all renegotiation in TLSv1.2 and earlier. Do not send HelloRequest
 messages, and ignore renegotiation requests via ClientHello.
 
+=item SSL_OP_IGNORE_UNEXPECTED_EOF
+
+Some TLS implementations do not send the mandatory close_notify alert on
+shutdown. If the application tries to wait for the close_notify alert but the
+peer closes the connection without sending it, an error is generated. When this
+option is enabled the peer does not need to send the close_notify alert and a
+closed connection will be treated as if the close_notify alert was received.
+
+Before OpenSSL 3.0, B<SSL_ERROR_SYSCALL> with errno=0 was reported in such cases.
+This behaviour contradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. When this option is enabled an B<SSL_ERROR_ZERO_RETURN> error is
+returned in case an EOF is detected and the close_notify alert was not
+received. You should only enable this option if the protocol running over TLS
```

Comment:
I think starting from "Before" until "received." can just be removed.

Reply:
Done. Thanks!